src/app/core/components/info-card/info-card.component.scss

```diff
@@ -0,0 +1,107 @@
+@import "../../../shared/variables/global.scss";
+
+// Variables
+$card-width: 28rem;
+$card-height: 15rem;
+$mario-logo-width: 200px;
+$mario-logo-height: 200px;
+$box-shadow: 0 8px 10px
+  rgba(
+    $color: #000000,
+    $alpha: 0.45,
+  );
+$letter-spacing-large: 5px;
+$font-size-huge: 110px;
+$letter-spacing-huge: 20px;
+
+// Mixins
+@mixin inner-container {
+  width: calc(#{$card-width} / 2);
+  display: flex;
+  justify-content: center;
+}
+
+@mixin font-style($weight, $size, $spacing, $color) {
+  font-family: $font-family;
+  font-weight: $weight;
+  font-size: $size;
+  letter-spacing: $spacing;
+  color: $color;
+}
+
+// Styles
+.card {
+  cursor: pointer;
+  height: $card-height;
+  border-radius: 5px;
```

Comment:
It could be variable as well

Reply:
3c32ffde529aea49d6bb39dc44bf0af94e69bae1